packages/headless-styles/src/components/Button/buttonCSS.scss

```
@@ -1,67 +1,83 @@
-.btnBase {
-  composes: pandoBtn from '../shared/button.module.css';
-}
-
-.btnDefaultBorder {
-  composes: pandoBtnBorder from '../shared/button.module.css';
-}
-
-.btnDefaultRadius {
-  composes: pandoBtnRadius from '../shared/button.module.css';
-}
+@use '../shared/button.scss' as *;
```

Using a wildcard namespace is a not a good engineering practice and should be kept to file globs IMHO. Let's update this to be more readable.

```suggestion
@use '../shared/button.scss' as shared;
```

---

Yeah, makes sense and I almost went back and updated them earlier.  I'll apply this to all of the cases where we're importing scss modules.